src/lib.rs

```diff
@@ -40,8 +41,24 @@ pub mod convert;
 #[cfg(feature = "std")]
 use std::vec::Vec;
 
-pub use crate::types::*;
 use crate::utils::*;
+use types::*;
```

Comment:
What's the diff between `crate::utils::*` and `types::*` ?

Reply:
None in this case. Which do you prefer?

Reply:
Using a single format throughout the project. Don't care which.

Reply:
In the top-level, we'll use the relative paths as it is easy shorthand. In general, though, if a submodule in this library wants to include something in a different branch of modules, it has to either use `super::` chains which are terrible or an absolute path.

Reply:
Got it.